frontend/src/components/ChatBox/ChatBox.tsx

```diff
@@ -235,7 +236,23 @@ function ChatBox({
 					</span>
 				</div>
 				<div className="control-buttons">
-					<Suspense fallback={<p>loading pdf library</p>}>
+					<Suspense
+						fallback={
+							<ThemedButton
+								className={'chat-button chat-button-disabled'}
+								onClick={() => {
+									console.error('This button is still loading');
+								}}
+								ariaDisabled={true}
+								tooltip={{
+									id: 'hey',
```

Comment:
This id must be unique in the document, so, best to use a meaningful id. In any case, I think we can do without a tooltip because it's expected to be a short-lived loading state. If the module fails to load then we have bigger (network) problems anyway.

Though in that case, it would be useful to keep your console.log statement indicating the button is loading, just in case. But then, make it a log, not an error.

Reply:
if we remove the tooltip there is nothing indicating to the user (sighted or on-sighted) why the button is disabled... Which might be okay but I think it's better to have a naff solution than no solution at all

Reply:
I've got it looking like this 
![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/a565cd10-df54-4789-a804-2d17561e182f)